### PR TITLE
bf: ZENKO-1585 Zenko userMD header zenko-instance-id

### PR DESCRIPTION
**Pull request by philipyoo**
Apply zenko user metadata header called "x-amz-meta-zenko-instance-id"
on the following operations that create objects or new versions
    - objectPut
    - multiObjectDelete
    - objectCopy
    - initiateMPU

Value of the header is the Zenko deployment hashed instance id

### CODE REVIEW

File: lib/api/apiUtils/object/applyZenkoUserMD.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const _config = require('../../../Config').config;
+
+/**
+ * applyZenkoUserMD - if request is within a Zenko deployment, apply user
+ * metadata called "zenko-source" to the object
+ * @param {Object} metaHeaders - user metadata object
+ * @return {undefined}
+ */
+function applyZenkoUserMD(metaHeaders) {
+    if (process.env.REMOTE_MANAGEMENT_DISABLE === '0' &&
```
Comment (JianqinWang) [inline lib/api/apiUtils/object/applyZenkoUserMD.js:12]: by default, isn't `REMOTE_MANAGEMENT_DISABLE` undefined and unspecified? Maybe it'd be better to check if someone explicitly set `REMOTE_MANAGEMENT_DISABLE=1` and work from there

**Review by JianqinWang — COMMENTED** [review]
just 1 quick comment, otherwise LGTM

**Comment by dora-korpar** [comment]
What about objectPutCopyPart? x)

File: lib/api/apiUtils/object/applyZenkoUserMD.js (hunk 1 above)
Comment (philipyoo) [reply lib/api/apiUtils/object/applyZenkoUserMD.js:12]: I think `REMOTE_MANAGEMENT_DISABLE` will only be equal to '0' when orbit is enabled. You're right about this being undefined for cases like running file or mem backend, but we can avoid setting this user md field in those cases

**Comment by philipyoo** [comment]
@dora-korpar I think only the initiateMPU is the one that needs to have the user md field set. It looked like putPart/copyPutPart ops will add to the mpu object created with initiateMPU, and on completeMPU, the user md headers defined on the mpu will be sent to the storage location

**Review by JianqinWang — APPROVED** [review]

**Comment by dora-korpar** [comment]
I somehow read objectPut as objectPutPart 🙃

**Review by dora-korpar — APPROVED** [review]

**Comment by philipyoo** [comment]
ty @JianqinWang and @dora-korpar :)

**Comment by philipyoo** [comment]
@bert-e approve

**Pull request merged**